src/ui/screens/help.rs

```diff
@@ -32,18 +32,25 @@ use crate::watched_tasks::WatchedTasksBuilder;
 const SCREEN_TYPE: AlertScreen = AlertScreen::Help;
 const PAGES: &[&str] = &[
     "Hey there!
-
 A short guide on how
 this interface works:
+
+Please long press the
+lower button to
+continue.
+...",
+    "...
+
 Long presses on the
-lower button perform
+lower button performs
 actions.
+
 ...",
     "...
 
 Short presses on the
-lower button toggle
-between actions.
+lower button toggles
+between options.
```

Comment:
I think `toggle` is correct here because it refers to the plural `presses` above?

Reply:
Correct. Thx!